pkg/controller/endpointslice/endpointslice_controller.go

```diff
@@ -302,20 +307,35 @@ func (c *Controller) syncService(key string) error {
 	return nil
 }
 
-// obj could be a *v1.Service or a DeletionalFinalStateUnknown marker item
+// obj could be a *v1.Service.
 func (c *Controller) enqueueService(obj interface{}) {
```

Comment:
Maybe this should be `onServiceUpdate`? The corresponding comment could be `onServiceUpdate updates the Service Selector in the cache and queues the Service for processing.`